bbschema/entity.py

```diff
@@ -343,10 +343,11 @@ def update(self, identifier_json):
 
         if 'value' in identifier_json:
             new.value = identifier_json['value']
-        if 'identifier_type_id' in identifier_json:
+
+        if 'identifier_type' in identifier_json:
```

Comment:
OK to change this, but I think we have a huge inconsistency across the WS about whether to accept objects containing an ID field, or whether to just accepts a flattened ID field.